### ISSUE TITLE
(Question) high-resolution rendering of segmented alpha-numeric displays

### ISSUE THREAD

**Issue by blautob** [markadr/dmd-viewer#7]
Hi @markadr ,

do you plan to add the  high-resolution rendering of segmented alpha-numeric displays as freezy pointed out here:
https://github.com/freezy/dmd-extensions/tree/master/LibDmd/Output/Virtual/AlphaNumeric

Is this even possible?

Thanks for your great work!

**Comment by markadr** [comment]
> do you plan to add the high-resolution rendering of segmented alpha-numeric displays as freezy pointed out here:
> https://github.com/freezy/dmd-extensions/tree/master/LibDmd/Output/Virtual/AlphaNumeric
> 
> Is this even possible?

Given it looks like these items are designed to ba placed in specific locations on a virutal backglass, not sure how it would work in the context of a remote DMD display - https://github.com/freezy/dmd-extensions/tree/master/LibDmd/Output/Virtual/AlphaNumeric#display-position

If these alpha-numeric displays are sent over the network as part of https://github.com/freezy/dmd-extensions/tree/master/Console/Server then it would in theroy be possible.

Do you have a good example table where these alpha-numeric displays are used?

**Comment by blautob** [comment]
You are completely right. I thought about it and indeed it actually doesn't make sense since those DMDs are to be placed on the Backglass itself.

However with having a look at all those beautiful looking DMDs i was wondering if it would be possible to use the app for the (new) glow effect features like discussed here https://www.vpforums.org/index.php?showtopic=46528

**Comment by markadr** [comment]
> However with having a look at all those beautiful looking DMDs i was wondering if it would be possible to use the app for the (new) glow effect features like discussed here https://www.vpforums.org/index.php?showtopic=46528

> How to make the DMDExt DMD look awesome with glow effects and slick DMDExt styles! (virtual DMD only)

As noted in the linked guide, these new effects will only be shown on the virtual DMD which I think means won't be sent to external displays or over the network, this sort of thing would likely need to be implemented on each remote display system.
